index.html

```diff
@@ -1003,6 +1001,545 @@ <h4><a href="#" class="link-dark nav-link p-0 mt-2"> Hokkaido: Japan’s Norther
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">FEATURES</span> CULTURE</h5>
+                            <p><a href="#" class="text-dark nav-link p-0 mt-2 fw-bold">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></p>
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">FEATURES</span> CULTURE</h5>
+                            <p><a href="#" class="text-dark nav-link p-0 mt-2 fw-bold">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></p>
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">Feature</span> OPINION</h5>
+                            <p><a href="#" class="text-dark nav-link p-0 mt-2 fw-bold">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></p>
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">PERISCOPE</span> CULTURE</h5>
+                            <p><a href="#" class="text-dark nav-link p-0 mt-2 fw-bold">FBI, DHS, White House Advisers Foresaw Possible Riots, Looked the Other Way</a></p>
+                        </div>
+                    </div>
+                    <div class="col-md-3 container-fluid px-3 py-3">
+                        <div class="mb-2">
+                            <a href="#"><img src="image/sin.jpg"  class="img-width" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5 class="fw-bold otext"><span class="first-header text-white">DOWNTIME</span> CULTURE</h5>
```

Comment:
- [ ] Titles also should have links in the `in the magazine` section.

Reply:
resolved :heavy_check_mark: